File: app/src/main/java/com/mcwilliams/TableTopicsApp/model/Category.java
package com.mcwilliams.TableTopicsApp.model;

/**
 * Created by m439047 on 5/29/13.
 */
public class Category {

    int _id;
    String _category;

    public Category() {
    }

    public Category(String category) {
        this._category = category;
    }

    public Category(int id, String category) {
        this._category = category;
        this._id = id;
    }

    public String get_category() {
        return _category;
    }

    public void set_category(String _category) {
        this._category = _category;
    }

    public int get_id() {
        return _id;
    }

    public void set_id(int _id) {
        this._id = _id;
    }
}

File: app/src/main/java/com/mcwilliams/TableTopicsApp/model/Category.kt
package com.mcwilliams.TableTopicsApp.model

import kotlin.properties.Delegates

/**
 * Created by m439047 on 5/29/13.
 */
class Category {

    var _id: Int = 0
    var _category: String by Delegates.notNull()

    constructor() {
    }

    constructor(category: String) {
        this._category = category
    }

    constructor(id: Int, category: String) {
        this._category = category
        this._id = id
    }
}
